ReactNative/ReactTwitter/core/views/tweetsListItem.js

```diff
@@ -36,10 +37,10 @@ var TweetsListItem = React.createClass({
             <View style={styles.textContainer}>
               <View style={styles.userInfoContainer}>
                 <Text style={styles.tweet_author}>{this.props.author_name}</Text>
-                <Text style={styles.tweet_author_handle}>@{this.props.author_handle}</Text>
+                <Text importantForAccessibility='no' style={styles.tweet_author_handle}>@{this.props.author_handle}</Text>
```

Comment:
yes exactly, these attributes are passes as properties for the component constructor, in this case `Text`